xtable-core/src/main/java/org/apache/xtable/avro/AvroSchemaConverter.java

```diff
@@ -160,11 +160,13 @@ private InternalSchema toInternalSchema(
           metadata.put(
               InternalSchema.MetadataKey.TIMESTAMP_PRECISION, InternalSchema.MetadataValue.MICROS);
         } else if (logicalType instanceof LogicalTypes.LocalTimestampMillis) {
-          newDataType = InternalType.TIMESTAMP_NTZ;
+          // TODO: Hudi 0.x writes INT64 in parquet, TimestampNTZType support added in 1.x
```

Comment:
Can you add a new issue tracking this and the below TODO and add the link in the comment?

Reply:
Done.